magicctapipe/scripts/lst1_magic/lst1_magic_create_irf.py

```diff
@@ -371,7 +371,7 @@ def create_irf(
 
     table_gamma, sim_info_gamma = load_dl2_data_file(input_file_gamma, quality_cuts, irf_type, dl2_weight)
 
-    if sim_info_gamma.viewcone.value != 0.0:
+    if sim_info_gamma.viewcone.value > 1.0:
```

Comment:
"Test" gamma ray MCs are produced with viewcone of 0.4 deg and were failing this check